src/lib/stark/ledger/createSignedTransfer.js

```diff
@@ -28,7 +28,7 @@ module.exports = async (
   const transport = await Transport.create()
   const eth = new Eth(transport)
   const tempKey = (await eth.starkGetPublicKey(starkPath)).toString('hex')
-  const starkPublicKey = {
+  let starkPublicKey = {
     x: tempKey.substr(2, 64),
     y: tempKey.substr(66)
   }
```

Comment:
Do we ever use tempKey for anything? Otherwise couldn't we add this inside starkGetPublicKey so that its directly returned in that format?